src/site/layouts/tests/liquid_template_axe_check_smoke_test/liquid_template_axe_check_smoke_test.unit.spec.js

```diff
@@ -0,0 +1,18 @@
+import { expect } from 'chai';
+import { parseFixture, renderHTML } from '~/site/tests/support';
+import axeCheck from '~/site/tests/support/axe';
+
+const layoutPath =
+  'src/site/layouts/liquid_template_axe_check_smoke_test.drupal.liquid';
+
+describe('liquid template axe check smoke test', () => {
+  const data = parseFixture(
+    'src/site/layouts/tests/liquid_template_axe_check_smoke_test/fixtures/liquid_template_axe_check_smoke_test.json',
```

Comment:
Is `liquid_template_axe_check_smoke_test` just there for example purposes?

Reply:
No, it's to make sure the Axe Checks are always working as expected. It was discovered this past week that Axe Checks were failing silently on the static pages so Trevor and I decided to add this smoke test to make sure they're always working. If this test ever fails, we'll know something is wrong with `axe-core` or something having to do with the Axe Checks.

Reply:
This is our positive affirmation the tests are running as expected. If they start failing silently for any reason, this test will act as a circuit breaker.